src/core/webidl.js

```diff
@@ -13,34 +13,33 @@ import { normalizePadding } from "core/utils";
 
 export const name = "core/webidl";
 
-var idlAttributeTmpl = tmpls["attribute.html"];
-var idlCallbackTmpl = tmpls["callback.html"];
-var idlConstTmpl = tmpls["const.html"];
-var idlDictionaryTmpl = tmpls["dictionary.html"];
-var idlDictMemberTmpl = tmpls["dict-member.html"];
-var idlEnumItemTmpl = tmpls["enum-item.html"];
-var idlEnumTmpl = tmpls["enum.html"];
-var idlExtAttributeTmpl = tmpls["extended-attribute.html"];
-var idlFieldTmpl = tmpls["field.html"];
-var idlIncludesTmpl = tmpls["includes.html"];
-var idlImplementsTmpl = tmpls["implements.html"];
-var idlInterfaceTmpl = tmpls["interface.html"];
-var idlIterableLikeTmpl = tmpls["iterable-like.html"];
-var idlLineCommentTmpl = tmpls["line-comment.html"];
-var idlMethodTmpl = tmpls["method.html"];
-var idlParamTmpl = tmpls["param.html"];
-var idlTypedefTmpl = tmpls["typedef.html"];
+const idlAttributeTmpl = tmpls["attribute.html"];
+const idlCallbackTmpl = tmpls["callback.html"];
+const idlConstTmpl = tmpls["const.html"];
+const idlDictionaryTmpl = tmpls["dictionary.html"];
+const idlDictMemberTmpl = tmpls["dict-member.html"];
+const idlEnumItemTmpl = tmpls["enum-item.html"];
+const idlEnumTmpl = tmpls["enum.html"];
+const idlExtAttributeTmpl = tmpls["extended-attribute.html"];
+const idlIncludesTmpl = tmpls["includes.html"];
+const idlImplementsTmpl = tmpls["implements.html"];
+const idlInterfaceTmpl = tmpls["interface.html"];
+const idlIterableLikeTmpl = tmpls["iterable-like.html"];
+const idlLineCommentTmpl = tmpls["line-comment.html"];
+const idlMethodTmpl = tmpls["method.html"];
+const idlParamTmpl = tmpls["param.html"];
+const idlTypedefTmpl = tmpls["typedef.html"];
 // TODO: make these linkable somehow.
 // https://github.com/w3c/respec/issues/999
 // https://github.com/w3c/respec/issues/982
-var unlinkable = new Set(["maplike", "setlike", "stringifier"]);
+const unlinkable = new Set(["maplike", "setlike", "stringifier"]);
 
 function registerHelpers() {
-  hb.registerHelper("extAttr", function(obj, indent) {
-    return extAttr(obj.extAttrs, indent, /*singleLine=*/ false);
+  hb.registerHelper("extAttr", function(obj) {
+    return extAttr(obj.extAttrs);
   });
   hb.registerHelper("extAttrInline", function(obj) {
```

Comment:
Maybe we even can kill `extAttrInline` and replace it by `extAttr`.

Reply:
extAttrInline was not actually used anywhere at all... so killing it.

Reply:
Actually, ignore that... it was used in a template.